Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3
```

Comment:
recommend that this is pegged to a specific minor version (probably 3.11)

Reply:
ill do this later meaning in 1 day or so